app/views/shared/_nav.html.erb

```diff
@@ -0,0 +1,73 @@
+<nav class="w-full bg-gray-800" data-controller="nav">
+  <div class="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
+    <div class="flex items-center justify-between h-16">
+      <div class="flex items-center">
+        <div class="flex-shrink-0">
+          <img class="h-8 w-8" src="https://tailwindui.com/img/logos/workflow-mark-indigo-500.svg" alt="Workflow">
+        </div>
+        <div class="hidden md:block">
+          <div class="ml-10 flex items-baseline space-x-4">
+            <!-- Current: "bg-gray-900 text-white", Default: "text-gray-300 hover:bg-gray-700 hover:text-white" -->
```

Comment:
maybe we can remove this?

Reply:
I guess it is good to keep it since it explains the CSS classes you need if you were to add more links in the navigation.